next.config.js

```diff
@@ -10,5 +10,4 @@ const nextConfig = {
 		API_KEY: 'cfe8f1e1a9b233b64412ec3cd0525b67',
```

Comment:
I am not a pro in `NextJS` but seeing an API key always makes me nervous. Usually the API keys are written in a `.env` file which would be added to the `.gitignore` so that no one can see the API key. 

But again, it might work differently with NextJS. 🤔

Reply:
Have a look at this article about environment variables in nextjs: https://nextjs.org/docs/basic-features/environment-variables